controllers/validationController.js

```diff
@@ -25,3 +27,36 @@ exports.validateNameAndEmail = [
     next();
     },
 ];
+
+exports.validateAdminPassword = (req, res, next) => {
+    if (process.env.NODE_ENV === 'production') {
+        console.log("It's in production_admin" );
+        adminPassword = process.env.ADMIN_PASSWORD;
```

Comment:
I think we can use `config` library for `ADMIN_PASSWORD` env var for better readability and the easier maintenance. 

Reference: https://github.com/nishi-ai/softball-register-web/issues/10#issue-982185592
I had written the explanation about `node-config` (yes, it was wrong to use on React but it will work perfectly on NodeJS)